modules/virt-understanding-vm-templates.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_templates/virt-creating-vm-template.adoc
+
+[id="virt-understanding-vm-templates-web_{context}"]
+= Understanding virtual machine templates
+
+Preconfigured Red Hat virtual machine templates are listed in the *Templates* tab within the *Virtualization* page. These  templates are available for different versions of Red Hat Enterprise Linux, Fedora, Microsoft Windows 10, and Microsoft Windows Servers. Each Red Hat virtual machine template is preconfigured with the operating system image, default settings for the operating system, flavor (memory), and workload type (server).
```

Comment:
flavor set cpu + memory, not just memory, sound be something like: flavor (cpu number, memory)

Reply:
Updated the content, please review.